__tests__/databaseSpec.jsx

```diff
@@ -20,40 +18,33 @@ describe('seedsData', () => {
 });
 
 describe('queryDB', () => {
-  test('it should return an empty result for nonExistant data', () => {
-    dbQuery.queryDB('select * from restaurant where id = 370', (err, res) => {
-      expect(res.length).toBe(1);
-      expect(res[0].title).toBeUndefined();
-    });
-  });
-  test('it should return a title, map and types property', () => {
-    dbQuery.queryDB('select * from restaurant where id = $1', [180], (err, res) => {
-      expect(res[0][1].length).to.be(1);
-    });
-  });
-  test('it should return data for current elements loaded in state', () => {
-    const wrapper = mount(<App />);
-    let title = wrapper.state('title');
-    dbQuery.queryDB('select * from restaurant where id = $1', [title.id], (err, res) => {
-      if (err) {
-        throw new Error(err);
-      }
-      expect(title.numstars).toEqual(res[0].numstars);
-    });
-  });
-  it('returns an error for a bad post value', () => {
+  test('it should return an empty result for nonExistant data', () => (
+    dbQuery.queryDB('select * from restaurant where id = ($1)', 370)
+      .then((res) => {
+        expect(res[0]).toBeUndefined();
+      })
+  ));
+
+  test('it should return a title, map and types property', () => (
+    dbQuery.queryDB('select * from restaurant where id = $1', 180)
+      .then((res) => {
+        expect(res[1].length).toBeGreaterThan(0);
+      })
+      .catch((err) => { throw new Error(err); })
```

Comment:
I think this could probably be refactored to drop the catch. The test would still fail if there were an error even without the catch, correct?